Makefile

```diff
@@ -25,9 +25,13 @@ clean:
 	helm delete orkestra -n orkestra 2>&1
 	kind delete cluster --name orkestra 2>&1
 
+ginkgo-test: install
+	go get github.com/onsi/ginkgo/ginkgo
+	ginkgo ./... -cover -coverprofile coverage.txt
```

Comment:
Out of curiousity, how does ginkgo CLI test runner different from go test? Is this something we should include in the development docs, that if you plan to run controller test use `ginkgo-test` target?

Reply:
`make test` should work for now; however, when we want to parallelize, `ginkgo` supports slightly different parallelization logic than `go test` has